mettle/api/api.go

```diff
@@ -72,17 +70,23 @@ func serve(opts grpcutil.Opts, name, requestQueue, responseQueue, preResponseQue
 		preResponses: common.MustOpenTopic(preResponseQueue),
 		jobs:         map[string]*job{},
 	}
+
 	go srv.Receive()
+	srv.jobs, err = getExecutions(opts, true)
```

Comment:
I would probably write this as
```
if jobs, err := getExecutions(opts, true); err != nil {
   ...
} else {
    srv.jobs = jobs
}
```
It's slightly more code but safer; it would be easy to return a nil map from `getExecutions`; in general you don't necessarily expect other values to be usable when your `err` is non-nil.